TAF/utils/scripts/docker/deploy-edgex.sh

```diff
@@ -37,13 +37,13 @@ else
 
     docker run --rm -v ${WORK_DIR}:${WORK_DIR}:rw,z -w ${WORK_DIR} -v /var/run/docker.sock:/var/run/docker.sock \
           --env WORK_DIR=${WORK_DIR} --env PROFILE=${PROFILE} --security-opt label:disable ${COMPOSE_IMAGE} \
-          -f "${WORK_DIR}/TAF/utils/scripts/docker/docker-compose.yaml" up -d edgex-proxy
+          -f "${WORK_DIR}/TAF/utils/scripts/docker/docker-compose.yaml" up -d proxy-setup
   fi
 
   docker run --rm -v ${WORK_DIR}:${WORK_DIR}:rw,z -w ${WORK_DIR} -v /var/run/docker.sock:/var/run/docker.sock \
           --env WORK_DIR=${WORK_DIR} --env PROFILE=${PROFILE} --security-opt label:disable ${COMPOSE_IMAGE} \
           -f "${WORK_DIR}/TAF/utils/scripts/docker/docker-compose.yaml" up \
-          -d database consul data metadata command notifications scheduler
+          -d database data metadata command notifications scheduler
```

Comment:
In theory, we also should startup redis `database` before `kong` as ADR indicating the startup sequence. However, if the `TAF tests` passed without time-out issues, I am ok with it.